src/Pelagos/Bundle/AppBundle/Resources/views/Menus/main.html.twig

```diff
@@ -47,10 +46,13 @@
                             <li class="last odd "><a href="https://data.gulfresearchinitiative.org/RB-data-compliance">Data Compliance Information</a></li>
                         </ul>
                     </li>
-                    <li class="last odd "><a href="http://research.gulfresearchinitiative.org/" title="GoMRI Research Information" name="GoMRI Research" target="_top" >Research</a></li>
+                    {% if is_granted('ROLE_DATA_REPOSITORY_MANAGER') %}
+                        {% include "PelagosAppBundle:Menus:admin.html.twig" %}
+                    {% else %}
+                        <li class="last odd"><a href="http://research.gulfresearchinitiative.org/" title="GoMRI Research Information" name="GoMRI Research" target="_top" >Research</a></li>
```

Comment:
I think this should be moved out the if-else, so that everyone can see this on the menu